main.go

```diff
@@ -426,7 +428,7 @@ func check_handler(w http.ResponseWriter, r *http.Request) {
 				}
 
 				c.Meta = ""
-				c.Receipt = string(tx.Amount) + "Payment received! It is safe to close the tab"
+				c.Receipt = strconv.FormatInt(tx.Amount, 10) + "Payment received! It is safe to close the tab"
 				c.Received = float64(tx.Amount) / 1000000000000
```

Comment:
You'll notice here I didn't use `strconv.FormatFloat`, because I haven't used it before but I wasn't sure what output you were expecting with the division of `1000000000000` as well. I'd rather admit ignorance than break your already working stuff.

Reply:
Thanks for these suggestions. The reason I manually `go get` the qrcode library is because it avoids the creation of the extra `go.mod` and `go.sum` files which I think are ugly. I'd also like to use as few libraries as possible so importing `strconv` to do the string conversion differently is a bit extra for me.
I super appreciate your suggestions as a more experienced programmer and have added some of them.

Reply:
> I super appreciate your suggestions

Yeah no problem!

> as a more experienced programmer

Phhhffbt. Tell that to my scrum master lol!

Jokes aside, yeah no worries. Take and leave what works for you. Based on your feedback, I didn't really understand your code's specific style and intent. So my suggestions _reaaaally_ weren't that good, but I'm glad they conveyed some signal even if they were mostly noise.